arcgis-ios-sdk-samples/Layers/Show labels on layers/README.md

```diff
@@ -40,4 +39,4 @@ Help regarding the JSON syntax for defining a label definition can be found in [
 
```

Comment:
hey @vquach2404 , I'm doing this same issue for Java and realised we also need to update the Additional information section which references the old JSON syntax approach. We'll need to update the design too, but I wondered if the following wording would be suitable:

## Additional information

Help regarding the Arcade label expression script for defining a label definition can be found on the (ArcGIS Developers)[https://developers.arcgis.com/arcade/] site.